### ISSUE TITLE
FidoAllianceCompliantMetadataService JWSVerifier initalization removes $jws content

### ISSUE THREAD

**Issue by Be-Mann** [web-auth/webauthn-framework#485]
### Version(s) affected

4.7.1

### Description

In the FidoAllianceCompliantMetadataService at the getJwsPayload at line 205, the JWSVerifier initalization removes the $jws, as well as the $token content, so the verifyWithKey naturally fails because this is null.

My goal is to initialize the metadata with FidoAllianceCompliantMetadataService, but at this point it makes $payloud = null.

### How to reproduce

```php
$MetadataService = FidoAllianceCompliantMetadataService::create(null, HttpClient::create(), 'https://mds3.fidoalliance.org/', [], null, 'https://valid.r3.roots.globalsign.com/');

$Aaguid = $authenticatorAttestationResponse->getAttestationObject()->getAuthData()->getAttestedCredentialData()->getAaguid() ?? null;
var_dump($MetadataService->getStatusReports($Aaguid->__toString()));
```

### Additional Context
[composer.json](https://github.com/web-auth/webauthn-framework/files/12703013/composer.json.txt)

**Comment by Be-Mann** [comment]
The var_dump is only to check if the data was loaded.

I am also open to suggestions for improving the loading/creation of metadata from the Fido Alliance Compliant Metadata Service data.

**Comment by Spomky** [comment]
Hi,

I am not sure to clearly understand this issue.

> the JWSVerifier initalization removes the $jws
The [line 205](https://github.com/web-auth/webauthn-framework/blob/4.7.1/src/metadata-service/src/Service/FidoAllianceCompliantMetadataService.php#L205) just create an instance of the service. It does rot affect the other variable.

**Comment by Be-Mann** [comment]
So I tried using FidoAllianceCompliantMetadataService to generate the current data into the metadata. On test I then call getStatusReports() which then executes getJwsPayload() via loadData().

And up to line 205 content also exists in the $jws, but after initializing JWSVerifier, the variable $jws, $token and $key = null, so in line 206 ``$isValid = $verifier->verifyWithKey($jws, $key, 0);`` nothing can be checked and so $payloud is then also = null.

In other words $MetadataService->getStatusReports($Aaguid->__toString()) == null, although the aaguid exists. To find out the problem I had once before and after line 205 a ``var_dump($jws);`` and noticed that after that the data is null, which had data before.

**Comment by Be-Mann** [comment]
And could it be reproduced? What is the current status?

**Comment by Spomky** [comment]
Hi,

JWSVerifier does not modifies the argument. There is no reason for having the data before initialization and not after.
Could you please use https://phpsandbox.io/ or similar and reproduce this behaviour?
